test/ButtonInputSpec.js

```diff
@@ -72,11 +72,11 @@ describe('ButtonInput', () =>{
     assert.notInstanceOf(result, Error);
   });
 
-  it('does not allow elements for children', function () {
+  it('allows elements as children', function () {
     ReactTestUtils.renderIntoDocument(
       <ButtonInput><span>blah</span></ButtonInput>
     );
 
-    shouldWarn('propType: Invalid');
+    console.warn.called.should.be.false;
```

Comment:
Thanks for the info, I'll remove. I picked up that pattern from line 46 of ButtonInputSpec, which I will remove as well.